protected/humhub/components/SettingsManager.php

```diff
@@ -134,6 +134,10 @@ public function isFixed(string $name): bool
      */
     public function getFixed(string $name)
     {
+        if ($value = $_ENV["{$this->moduleId}_{$name}"] ?? null) {
```

Comment:
We can probably remove this with the automatic mapping between ENV and Config, right?